src/components/tabs/Tabs.vue

```diff
@@ -33,6 +37,15 @@
           }
         }
         return tabs
+      },
+      styleWidth() {
+        if (!this.fullWidth) return
+
+        const width = `${100 / this.tabs.length}%`
+        return {
+          width,
```

Comment:
Tab headers are already in a flex, it's enough to pass `flex: 1` instead of a width here. It also makes them scale properly when one tab is longer or when they occupy more than one line.